spec/acceptance/f5_profileclientssl/rest_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper_acceptance'
+
+describe 'f5_profileclientssl' do
+  it 'creates and updates serverssl-profile1' do
+    pp=<<-EOS
+    f5_profileclientssl {'/Common/clientssl-profile1':
+       ensure                          => 'present',
+       cert                            =>"/Common/default.crt",
+       key                             =>"/Common/default.key",
+       proxy_ssl                        => 'enabled',
+       proxy_ssl_passthrough           => 'enabled',
+    }
+    EOS
+    make_site_pp(pp)
+    run_device(:allow_changes => true)
+
```

Comment:
Need check for idempotency.

Reply:
Update for both f5_profileclientssl and f5_profileserverssl